pyuvdata/parameter.py

```diff
@@ -84,7 +85,7 @@ class UVParameter(object):
         A fake value that can be assigned to a non-required UVParameter if the
         metadata is required for a particular file-type.
         This is not an attribute of required UVParameters.
-    form : 'str' or tuple
+    form : 'str', int or tuple
         Either 'str' or a tuple giving information about the expected
```

Comment:
we should probably add int to the description here too.

Reply:
yep. I realized this could be an int (we have existing examples and code to handle it being an int) and added it here with the intention of propagating it to the other places but forgot, thanks for catching it!